order_service_test.go

```diff
@@ -87,6 +164,18 @@ func (s *orderServiceTestSuite) assertCreateOrderResponseEqual(e, a *CreateOrder
 	r.Equal(e.TimeInForce, a.TimeInForce, "TimeInForce")
 	r.Equal(e.Type, a.Type, "Type")
 	r.Equal(e.Side, a.Side, "Side")
+
+	for idx, fill := range e.Fills {
+		s.assertFillEqual(fill, a.Fills[idx])
```

Comment:
`a.Fills[idx]` may cause out of index error, it's better to assert `r.Len(a.Fills, len(e.Fills))` before `range`